pkg/volume/plugins.go

```diff
@@ -42,6 +42,13 @@ type VolumeOptions struct {
 	RootContext string
 }
 
+// System is an interface that abstracts certain operations on a host operating
+// system that a volume might want to execute.
+type System interface {
```

Comment:
To be honest I just followed the same path as mount interface. When reading Host interface I was under the impression that it's supposed to be used only to talk to Kubelet not the host machine itself. Do you suggest moving both interfaces to Host?